source/acquaman/AMGenericScanActionControllerAssembler.cpp

```diff
@@ -124,16 +124,28 @@ AMAction3* AMGenericScanActionControllerAssembler::generateActionTreeForStepAxis
 		// generate axis initialization list
 		AMListAction3 *initializationActions = new AMListAction3(new AMListActionInfo3(QString("Initializing %1").arg(axisControl->name()), QString("Initializing Axis with Control %1").arg(axisControl->name())), AMListAction3::Sequential);
 
-		AMAction3 *initializeControlPosition = AMActionSupport::buildControlMoveAction(axisControl, stepScanAxis->axisStart());
+		AMAction3 *initializeControlPosition = 0;
+
+		if (direction_ == AMScanConfiguration::Increase)
+			initializeControlPosition = AMActionSupport::buildControlMoveAction(axisControl, stepScanAxis->axisStart());
```

Comment:
No, `axisStart` and `axisEnd` should always (ideally) be the way they always have (i.e.: -10 to 10) and it's up to the direction to build the actions such that the direction constraint is satisfied.  I'm sure there is probably a better way but this is what we'll use for version 1.